apps/order.rb

```diff
@@ -0,0 +1,54 @@
+RodaApp.route('order') do |r|
+
+  # POST /api/orders/new
+  r.post 'new' do
+    message = CreateOrder.new(r.params).call
+    PostToSlack.say(:food_bot, message) unless test?
+    status(200)
+  end
+
+  # POST /api/orders/show
+  r.post 'show' do
+    order_list = OrderList.last
+    orders = order_list.show_orders
+    if order_list.closed
+      PostToSlack.say(:food_bot, "Orders are closed... Anyway last ones: \n #{ orders }") unless test?
+    else
+      PostToSlack.say(:food_bot, "Check if its okey: \n #{ orders }") unless test?
+    end
+    status(200)
+  end
+
+  # POST /api/orders/close
+  r.post 'close' do
+    order_list = OrderList.last
+    orders = order_list.show_orders
+    if order_list.closed
+      PostToSlack.say(:food_bot, "Already closed") unless test?
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.